packages/cerebral-provider-firebase/src/remove.js

```diff
@@ -1,13 +1,14 @@
 import {
   createRef
 } from './helpers'
+import {FirebaseProviderError} from './errors'
 
 export default function remove (path) {
   const ref = createRef(path)
-  return new Promise((resolve, reject) => {
-    ref.remove().then(
-      () => resolve({}),
-      (error) => reject({error: error.message})
-    )
-  })
+
+  return ref.remove()
+    .then(() => undefined)
```

Comment:
You still don't get it right.
`() => {}` doesn't return anything {} is just a block.
`() => ({})` returns an object
`() => undefined` returns undefined  which can be accidentally become defined in outer/global scope